Tests/NostrSDKTests/Events/DeletionEventTests.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  DeletionEventTests.swift
+//  
+//
+//  Created by Terry Yiu on 4/14/24.
+//
+
+@testable import NostrSDK
+import XCTest
+
+final class DeletionEventTests: XCTestCase, EventCreating, EventVerifying, FixtureLoading {
```

Comment:
I noticed that there's a missing test for decoding deletion events. We can add that separately.